lib/food.js

```diff
@@ -56,17 +56,59 @@ const Food = class Food {
       });
     };
   };
+
+  editName() {
+    const toEdit = {name: $(event.target).val(), calories: $(event.target).parent().next().text()}
+
+    const tr = $(event.target).parent().parent()
+    const id = tr.data('food-id')
+
+    $.ajax({
+      url: API + `foods/${id}`,
+      method: 'PATCH',
+      data: toEdit
+    })
+    .then( (data) => {
+      updateCellName(data, tr)
+    })
+    .fail( (error) => {
+      console.error(error)
+    });
+  };
+
+  editCalories() {
+    const toEdit = {name: $(event.target).parent().prev().text(), calories: $(event.target).val()}
+
+
+    const tr = $(event.target).parent().parent()
+    const id = tr.data('food-id')
+    // console.log(tr)
+
+    $.ajax({
+      url: API + `foods/${id}`,
+      method: 'PATCH',
+      data: toEdit
+    })
+    .then( (data) => {
+      updateCellCalories(data, tr)
+    })
+    .fail( (error) => {
+      console.error(error)
+    });
+  };
 };
 
+
+
 function makeFoodTable(data) {
   data.forEach( (food) => {
-    $('.foods tbody').prepend(`<tr><td >${food.name}</td><td>${food.calories}</td><td><button data-food-id=${food.id} class='remove btn red darken-3'>-</button></td></tr>`)
+    $('.foods tbody').prepend(`<tr data-food-id=${food.id}><td class='food-cell' >${food.name}</td><td class='calorie-cell'>${food.calories}</td><td><button data-food-id=${food.id} class='remove btn red darken-3'>-</button></td></tr>`)
```

Comment:
The string interpolation here makes this a lot easier to read. Thanks for taking this route.